addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -69,3 +69,11 @@
         object-fit: contain !important;
     }
 }
+
+// Override the default image selection color used in "web_editor". This
+// prevents selected images from being covered with a blue filter in edit mode
+// (Website, Email Marketing). This allows to see the correct final rendering
+// of an image when it is edited.
+.odoo-editor-editable img::selection {
```

Comment:
I tested and on the mail marketing app, when I create a new mail template and I choose the first "type" (plain text), I do `/image`, I import an image, I click on it:
On your branch: no blue filter
On 16.0: a blue filter is good

Since in this scenario there are no options for images, having a blue filter on the image makes sense (at least as much as when you edit a task in a project)

I admit that the scenario is a bit tricky but I'm sure you can make it work perfectly :+1:

Reply:
@Guillaume-gdi ok changes are done